### DISCUSSION TITLE
How to save checkpoints and load optimizers in each checkpoints.

### DISCUSSION THREAD

**Discussion by Navanit-git** [karpathy/build-nanogpt#76] [Q&A]
Hi I have a small GPU and I want to save or pre train the model using checkpoints. So can anyone guide me also in the video at 03:53:52 seconds, Andrej has explained a bit but how to save and reload the model for next checkpoint iteration. I am new to this, so if anyone can help. Thankyou

**Comment by PingleSlayer** [comment]
You should take a look at the train.py file in [karpathy/nanoGPT](https://github.com/karpathy/nanoGPT).
